### ISSUE TITLE
"Do not prefix reader method names with get_" shouldn't apply to a function outside of a class

### ISSUE THREAD

**Issue by ianfixes** [rubocop-hq/rubocop#4233]
Sometimes it makes sense to begin a function name with `get`, particularly when dealing with IO.  This is noted here: https://github.com/bbatsov/ruby-style-guide/issues/423

For functions that are not part of classes, such usage would not conflict with the spirit of this rule -- they are not accessors.  `get_char`, for instance.

--------

## Expected behavior

Rubocop would not apply a rule about "accessors" to a class if there is no actual class.  Alternately, the rule would have an option to ignore standalone methods.

## Actual behavior

Rule is unconditionally applied to methods outside classes.

## Steps to reproduce the problem

Rubocop finds fault with the following:

```ruby
# http://stackoverflow.com/a/14527475/2063546
def get_char
  state = `stty -g`
  `stty raw -echo -icanon isig`

  STDIN.getc.chr
ensure
  `stty #{state}`
end
```

## RuboCop Version

OSX 10.12.3

```
$ rubocop -v
0.46.0
```

**Comment by bbatsov** [comment]
> Rubocop would not apply a rule about "accessors" to a class if there is no actual class. Alternately, the rule would have an option to ignore standalone methods.

I don't see why it's more likely to have a valid usecase for this in a standalone method then in a class. There are always going to be some valid usecases for this and it such cases I'd simply suggest using a disable directive.

**Comment by ifreecarve** [comment]
Fair enough.  I'm reporting this specifically because the rule talks about "accessors", and in this case there is no class in which to access something.

**Comment by bbatsov** [comment]
Hmm, when you put it like this... Maybe it makes some sense to tweak this cop.

**Comment by bbatsov** [comment]
There hasn't been much activity on this ticket and our Core Team is spread too thin on so many tasks, so I'll just close it for the sake of having a cleaner lists of tasks to focus on. We'd gladly review a PR, but it's unlikely that'd we're going to tackle this ourselves in the foreseeable future.

**Comment by ianfixes** [comment]
Do you have a label for "contributions welcome" that you could apply here instead of closing it?  The issue itself hasn't gone away, after all
